### PR TITLE
docs(connector): wechat web readme zh-CN

### PR DESCRIPTION
**Pull request by darcyYe**
## Summary

Add WeChat web readme zh-CN contents.

## Testing

N/A

### CODE REVIEW

File: packages/connector-wechat/README.md [hunk 1]
```diff
@@ -75,3 +85,65 @@ Once passed the review, go to the application details page and generate an AppSe
...
 
 Once WeChat web connector is enabled, you can sign in to your app again to see if it works.
+
+# 微信网页连接器
+
+## 开始上手
+
+如果你还不知道连接器的概念，或者还没有将本连接器添加至你的「登录体验」，请先参见 [Logto 教程](https://docs.logto.io/zh-cn/docs/tutorials/get-started/enable-social-sign-in)。
+
+
```
Comment (IceHe) [inline packages/connector-wechat/README.md:95]:
```suggestion

```

File: packages/connector-wechat/README.md [hunk 2]
```diff
@@ -75,3 +85,65 @@ Once passed the review, go to the application details page and generate an AppSe
...
+> **⚠️ Caution**
+> 
+> 这个连接器只适用于网页应用。如果你在寻找移动端原生应用的微信登录解决方案，请移步 [微信原生连接器](https://github.com/logto-io/logto/tree/master/packages/connector-wechat-native)。
+
+## 在微信开放平台新建一个网页应用
+
+> 💡 **Tip**
+> 
+> 如果你已经完成了某些步骤可以选择直接跳过。
```
Comment (gao-sun) [inline packages/connector-wechat/README.md:104]: should we align with `wechat-native`?

File: packages/connector-wechat/README.md [hunk 3]
```diff
@@ -75,3 +85,65 @@ Once passed the review, go to the application details page and generate an AppSe
...
+
+### 创建一个网页应用
+
+用刚创建的帐号登录。在「移动应用」标签页，点按大而绿的「创建移动应用」按钮。
+
+![App tabs](/packages/connector-wechat/docs/app-tabs.png)
+
+让我们填写一下申请表单里的必要信息。
+
+![Create a mobile app](/packages/connector-wechat/docs/create-web-app.png)
```
Comment (gao-sun) [inline packages/connector-wechat/README.md:118]: Creat a web app

Comment (gao-sun) [reply packages/connector-wechat/README.md:118]: same for the eng version

File: packages/connector-wechat/README.md [hunk 4]
```diff
@@ -75,3 +85,65 @@ Once passed the review, go to the application details page and generate an AppSe
...
+
+#### 基础信息
+
+大多数字段都很直接。在完成了表格之后，点按「下一步」以继续。
+
+#### 网站信息
+
+用你的 Logto 域（例如 `logto.io`）填写「授权回调域」。
```
Comment (gao-sun) [inline packages/connector-wechat/README.md:126]:
```suggestion
用你的 Logto 域名（例如 `logto.io`）填写「授权回调域」。
```

**Review by gao-sun — APPROVED** [review]

File: packages/connector-wechat/README.md (hunk 2 above)
Comment (darcyYe) [reply packages/connector-wechat/README.md]: aligned.

File: packages/connector-wechat/README.md (hunk 3 above)
Comment (darcyYe) [reply packages/connector-wechat/README.md]: fixed.

**Pull request merged**